pkg/strategy/dca2/dev_mode.go

```diff
@@ -0,0 +1,7 @@
+package dca2
+
+// DevMode, if Enabled is true. it means it will check the running field
+type DevMode struct {
+	On            bool `json:"on"`
+	IsNewStrategy bool `json:"isNewStrategy"`
```

Comment:
how about `newAccount` ?

Reply:
Actually, it's not new account. This field is used to make dca2 strategy not recover with long time ago trades.